site/content/docs/testapps/crapi.md

```diff
@@ -15,6 +15,46 @@ It is open source, and maintained by OWASP and the community. It’s fairly well
 * Repo - https://github.com/OWASP/crAPI
 * Docker - https://hub.docker.com/u/crapi
 
+### Quick Start
+
+New to ZAP and just want to quickly run ZAP against crAPI?
+
+Just run these commands:
+
+```bash
+# Creating and using a directory called "crapi" is important, dont skip this step!
+mkdir crapi
+cd crapi
+curl -o docker-compose.yml https://raw.githubusercontent.com/OWASP/crAPI/refs/heads/main/deploy/docker/docker-compose.yml
+
+# Start crapi
+docker-compose up
+
+# Download the recommended plan using curl, or use any other suitable tool
+curl -O https://raw.githubusercontent.com/zaproxy/community-scripts/refs/heads/main/other/af-plans/FullScanCrApiAuth.yaml
+
+# Run ZAP using the stable Docker image, the crapi docker network, 
+# mapping the CWD so that Docker can access the file and export the report.
+# The network name is based on the cwd name, which is why creating and using "crapi" is important
+docker run --network crapi_default -v $(pwd):/zap/wrk/:rw -t zaproxy/zap-stable zap.sh -cmd -autorun wrk/FullScanCrApiAuth.yaml
+```
+
+Note that the yaml file includes the requests needed to register the user used for authentication.
+
+To run this command on Windows see the [relevant documentation](/docs/docker/about/#mounting-the-current-directory).
+
+You will need to have Docker installed. If you do not want to use Docker then you can of course install both crAPI and ZAP locally.
+
+This command should take around TBA minutes (once the Docker image has been downloaded) on a MacBook M2 and should find the following Medium risk alerts:
+
+* 🟠 [.env Information Leak](/docs/alerts/40034/)
+* 🟠 [CORS Misconfiguration](/docs/alerts/40040/)
+* 🟠 [Content Security Policy (CSP) Header Not Set](/docs/alerts/10038/)
+* 🟠 [Cross-Domain Misconfiguration](/docs/alerts/10098/)
+* 🟠 [Missing Anti-clickjacking Header](/docs/alerts/10020/)
+
+It will create an HTML file in your CWD containing full details of all of the issues found.
```

Comment:
I know this is the text we used on the one yesterday but should we say "report" instead of "file"?